theano/compile/function_module.py

```diff
@@ -752,9 +752,35 @@ def checkSV(sv_ori, sv_rpl):
         return f_cpy
 
     def __call__(self, *args, **kwargs):
+        """
+        Evaluates value of a function on given arguments.
+
+        Parameters
+        ----------
+        args : list
+            List of inputs to the function. All inputs are required, even when
+            some of them are not necessary to calculate requested subset of
+            outputs.
+
+        kwargs : dict
+            TODO: other kwargs?
```

Comment:
User can pass inputs by kwargs. The key can be the input variable name or instance. What about appending something like this to your first paragraph and removing this todo:

The function inputs can be passed as keyword argument. For this, use the name of the input or the input instance as the key.